Transports/com.community.netcode.transport.websocket/Runtime/WebSocketTransport.cs

```diff
@@ -122,9 +125,15 @@ public override bool StartServer()
             {
                 throw new InvalidOperationException("Socket already started");
             }
-
-            WebSocketServer = new WebSocketServer(Port);
+            
+            WebSocketServer = new WebSocketServer(Port, SecureConnection);
```

Comment:
I'm actually quite surprised how this worked before. WebSocketServer is a class but we're never initializing it to a different value than null?